CMakeLists.txt

```diff
@@ -49,26 +55,57 @@ add_definitions(${DBusCpp_CFLAGS_OTHER})
 add_definitions(${Glibmm_CFLAGS_OTHER})
 add_definitions(${LXC_CFLAGS_OTHER})
 add_definitions(${Jansson_CFLAGS_OTHER})
+add_definitions(-DINSTALL_PREFIX="${CMAKE_INSTALL_PREFIX}")
 
 include_directories(${IVILogging_INCLUDE_DIRS})
 include_directories(${DBusCpp_INCLUDE_DIRS})
 include_directories(${Glibmm_INCLUDE_DIRS})
 include_directories(${LXC_INCLUDE_DIRS})
 include_directories(${Jansson_INCLUDE_DIRS})
 
+include_directories(
+    common
+    libsoftwarecontainer/src
+    libsoftwarecontainer/include
+    ${CMAKE_CURRENT_BINARY_DIR}
+    ${CMAKE_CURRENT_BINARY_DIR}/libsoftwarecontainer/src
+)
+
 link_directories(${IVILogging_LIBRARY_DIRS})
 link_directories(${DBusCpp_LIBRARY_DIRS})
 link_directories(${Glibmm_LIBRARY_DIRS})
 link_directories(${LXC_LIBRARY_DIRS})
 link_directories(${Jansson_LIBRARY_DIRS})
 
+#
+# All available options goes here
+#
+
+# First, options on enabling / disabling gateways at compile-time
```

Comment:
I'm not a fan of the "first", "then", style. I've seen us do it before and to me it doesn't bring much more than overhead if things are moved around at some point.